astropy/time/core.py

```diff
@@ -1787,6 +1787,85 @@ def light_travel_time(self, skycoord, kind='barycentric', location=None, ephemer
         tcor_val = (spos * cpos).sum(axis=-1) / const.c
         return TimeDelta(tcor_val, scale='tdb')
 
+    def to_local(self, theta, longitude):
```

Comment:
I think this should be a private method, so `_to_local`, since I don't think we want direct use of this.

Reply:
Sure.